model/relationship.js

```diff
@@ -0,0 +1,28 @@
+module.exports = function (sequelize, DataTypes) {
+    var Relationship = sequelize.define("Relationship", {
+        relationshipType: {
+            type: DataTypes.STRING,
+            allowNull: false,
+        },
+        relatingUserId: {
+            type: DataTypes.STRING,
+            allowNull: false
+        },
+        relatedUserId: {
+            type: DataTypes.STRING,
+            allowNull: false
+        },
+    });
+
+    // Post.associate = function (models) {
+    //     // We're saying that a Post should belong to an Author
+    //     // A Post can't be created without an Author due to the foreign key constraint
+    //     Post.belongsTo(models.Author, {
```

Comment:
```suggestion
    //     Relationship.belongsTo(models.User, {
```

Reply:
I am not sure how this ought to change considering that it's not exactly a foreign key; it's two users from the same table relating to each other. ><